hosting/single/runner.sh

```diff
@@ -97,10 +97,12 @@ fi
 sleep 10
 
 pushd app
-pm2 start -l /dev/stdout --name app "yarn run:docker"
+pm2 start --name app "yarn run:docker"
```

Comment:
What does this change - does this mean that the pm2 logs will come out of the main docker logging?

Reply:
As far as I can tell this flag was doing nothing.